CONTRIBUTING.md

```diff
@@ -1,3 +1,4 @@
 # Contributing to the JDK
 
-Please see <https://openjdk.org/contribute> for how to contribute.
+Please see the [OpenJDK Developers’ Guide](https://openjdk.org/guide/),
+especially the section [Working With Pull Requests](https://openjdk.org/guide/#life-of-a-pr).
```

Comment:
I'm not sure about the second line. "Contributing" can mean a wide variety of things. I think we should leave it to the guide to break this down for any new user.